starters/storybook/app/components/blocks/CardGroup/CardGroup.tsx

```diff
@@ -0,0 +1,116 @@
+import { cva, type VariantProps } from 'class-variance-authority'
+import { ComponentPropsWithoutRef } from 'react'
+import {
+  Button,
+  ButtonProps,
+  SimpleCard,
+  SimpleCardProps,
+  TeaserCard,
+  TeaserCardProps,
+} from '~/components/primitives'
+import { cn } from '~/lib/utils'
+
+const cardGroupVariants = cva('w-full py-12 md:py-16', {
+  variants: {},
+  defaultVariants: {},
+})
+
+type CardItem = SimpleCardProps | TeaserCardProps
+
+export interface CardGroupProps
+  extends ComponentPropsWithoutRef<'div'>,
+    VariantProps<typeof cardGroupVariants> {
+  heading?: string
+  subheading?: string
+  description?: string
+  action?: ButtonProps
+  cards: CardItem[]
+}
+
+export const CardGroup = ({
+  className,
+  heading,
+  subheading,
+  description,
+  action,
+  cards,
+  ...props
+}: CardGroupProps) => {
+  return (
+    <div className={cn(cardGroupVariants(), className)} {...props}>
+      <div className="container mx-auto">
+        <div className="mb-12 text-center">
+          {subheading && (
+            <h5 className="mb-4 text-lg font-semibold text-gray-900 dark:text-gray-100 sm:text-xl md:text-2xl">
+              {subheading}
+            </h5>
+          )}
+          <h2 className="mb-5 text-3xl font-bold text-gray-900 dark:text-gray-100 sm:text-4xl md:text-5xl">
+            {heading}
```

Comment:
Heading is required? In the typing is 
```ts
heading?: string
```

but I don't see any guard if is not passed, or the property is required?